com.unity.render-pipelines.high-definition/Runtime/Lighting/PlanarReflectionFiltering.compute

```diff
@@ -0,0 +1,200 @@
+#pragma kernel FilterPlanarReflection
+#pragma kernel DownScale
+#pragma kernel DepthConversion
+
+#pragma only_renderers d3d11 playstation xboxone vulkan metal switch
```

Comment:
More for another PR, but ideally we would like to run this filtering async.
i.e during the prepass  /decal... I will keep track of this.

Reply:
ajout to explain algorithm

Reply:
done for the explanation